src/main/kotlin/blackjack/Card.kt

```diff
@@ -1,3 +1,9 @@
 package blackjack
 
-data class Card(val number: CardNumber, val suit: Suit)
+data class Card(val number: CardNumber, val suit: Suit) : Comparable<Card> {
+    fun number(): Int {
+        return number.baseValue
+    }
+
+    override fun compareTo(other: Card): Int = number.compareTo(other.number)
```

Comment:
소소하지만 compareTo는 infix 함수이기에 다음과 같이 dot을 생략할수도 있습니다 ㅎㅎ

```suggestion
    override fun compareTo(other: Card): Int = number compareTo other.number
```